tests/query/snql/test_query.py

```diff
@@ -535,6 +561,112 @@
         ),
         id="Basic query with new lines and no ambiguous clause content",
     ),
+    pytest.param(
+        """MATCH (e:events)
+        SELECT 4-5,3*foo(c) AS foo,c
+        WHERE or(equals(arrayJoinExplode(a, '=', 'RuntimeException'), 1), equals(arrayJoinExplode(b, 'NOT IN', tuple('Stack', 'Arithmetic')), 1)) = 1""",
```

Comment:
Is there a reason to use the prefix notation in these tests? I remember SnQL worked with the infix notation. OR is it just personal preference ?

```
WHERE arrayJoinExplode() = 1 or arrayJoinExplode(...) = 1
```

Reply:
This one is like this because I copied it from a discover test, and the discover tests all use prefix notation for `or` conditions to get around the JSON schema limitations. It could be infix as well.